### PR TITLE
Add attribute `sharing/uri` and conditions `costs_relevant` and `long_term_preservation`

### PR DESCRIPTION
**Pull request by Zack-83**
Proposed by #162 and discussed in the 24/11/2022 meeting

Also sorted some attributes and conditions

### CODE REVIEW

File: rdmorganiser/conditions/rdmo.xml [hunk 1]
```diff
@@ -27,6 +9,15 @@
 		<target_text/>
 		<target_option dc:uri="https://rdmorganiser.github.io/terms/options/other_requirements_options/146"/>
 	</condition>
+	<condition dc:uri="https://rdmo.mathplus.de/terms/conditions/costs_relevant">
+		<uri_prefix>https://rdmo.mathplus.de/terms</uri_prefix>
```
Comment (Zack-83) [inline rdmorganiser/conditions/rdmo.xml:13]:
```suggestion
	<condition dc:uri="https://rdmorganiser.github.io/terms/conditions/costs_relevant">
		<uri_prefix>https://rdmorganiser.github.io/terms</uri_prefix>
```

File: rdmorganiser/conditions/rdmo.xml [hunk 2]
```diff
@@ -72,6 +72,15 @@
 		<target_text>1</target_text>
 		<target_option/>
 	</condition>
+	<condition dc:uri="https://rdmo.mathplus.de/terms/conditions/long_term_preservation">
+		<uri_prefix>https://rdmo.mathplus.de/terms</uri_prefix>
+		<key>long_term_preservation</key>
```
Comment (Zack-83) [inline rdmorganiser/conditions/rdmo.xml:77]:
```suggestion
	<condition dc:uri="https://rdmorganiser.github.io/terms/conditions/long_term_preservation">
		<uri_prefix>https://rdmorganiser.github.io/terms</uri_prefix>
		<key>long_term_preservation</key>
```

**Comment by thrichter-FD** [comment]
Fine with me - after the changes addressed by Zack-83.

**Review by jmfrenzel — APPROVED** [review]
Thank's. This looks good to me.

**Review by thrichter-FD — APPROVED** [review]

**Pull request merged**